slackclient/_server.py

```diff
@@ -123,12 +123,34 @@ def send_to_websocket(self, data):
         """
         try:
             data = json.dumps(data)
-            self.websocket.send(data)
+            return self.websocket.send(data)
         except:
             self.rtm_connect(reconnect=True)
+        return None
 
-    def ping(self):
-        return self.send_to_websocket({"type": "ping"})
+    def ping(self, pingid=None, **kwargs):
+        """ Send a ping message over the websocket. ping accepts any
+        key:value: pair and returns those values with the pong response.
+
+        :Args:
+            pingid (Optional[int]): The id of the ping. The reply_to
+                value of the response pongwill use this value. This
+                argument takes priority of the id key in kwargs, if
+                present.
+        :Kwargs:
+            (Optional) Any kwy:value pairs to be included in the ping.
+
+        :Example:
+            sc.server.ping(1234, timestamp=time.time())
+
+        :Returns:
+            (id, bytes_sent) tuple. If bytes_sent is None, the send
+                failed.
+        """
+        if pingid:
```

Comment:
If we keep it as `id`, it'd clean this up a little bit, too :)

Reply:
Just to clarify, do you still want the positional argument? I could just drop it and not do any special `id` handling, it would just be included in `kwargs`.